pkg/asset/machines/master.go

```diff
@@ -567,6 +567,12 @@ func (m *Master) Generate(ctx context.Context, dependencies asset.Parents) error
 				return errors.Wrap(err, "failed to create ignition for custom NTP for master machines")
 			}
 			machineConfigs = append(machineConfigs, ignChrony)
+
+			ignRoutes, err := machineconfig.ForExtraRoutes("master", []string{"166.8.0.0/14", "161.26.0.0/16"}, ic.MachineNetwork[0].CIDR.String())
```

Comment:
Same here, we should store these somewhere with a name that explains their purpose.

Reply:
PTAL the latest force-push. Thx.